Documentation/prerequisites-for-building.md

```diff
@@ -1,5 +1,11 @@
 The following pre-requisites need to be installed for building the repo:
 
+# Install .NET Core 2.1 SDK
```

Comment:
I would change this to .NET Core 3.1 SDK now that it shipped. There is no good reason to stick with the old version.